inference/enterprise/workflows/entities/steps.py

```diff
@@ -432,31 +511,44 @@ def validate_field_binding(self, field_name: str, value: Any) -> None:
 
 
 class InstanceSegmentationModel(ObjectDetectionModel):
```

Comment:
Is there an easy way to know from the manifest whether something is a subclass of BaseModel? I think right now I just check if the name ends in `_model`

Reply:
do you mean `BaseModel` or `RoboflowModel`. 
If the former - each manifest class must be derived after `pydantic.BaseModel`
If the latter - no way now - I feel like additional metadata about steps can be held in
```
model_config = ConfigDict(
        json_schema_extra={
            "description": "This block represents inference from Roboflow multi-class classification model.",
            "docs": "https://inference.roboflow.com/workflows/classify_objects", <- additional keys here
        }
    )
```

Reply:
Is that in the manifest? I don't see `BaseModel` or `base_model` in there anywhere.

Reply:
no it is not, that's just required for pydantic to work